client/src/components/BookList/BookListShowPage.js

```diff
@@ -1,47 +1,67 @@
 import React, { useState, useEffect } from "react";
 import { withRouter } from "react-router-dom";
 import BookTile from "../Book/BookTile.js";
-
+import BookListIndex from "../BookList/BookListIndex.js";
 const showBooksBookList = (props) => {
   const [books, setBooks] = useState([]);
   const [bookListName, setBookListName] = useState("");
-  const [bookFavorites, setBookFavorites] = useState("");
+  const [bookLists, setBookLists] = useState([]);
   const bookListId = props.match.params.id;
 
-  const getBooks = async () => {
+  const getBookListName = async () => {
     try {
-      const response = await fetch(`/api/v1/bookLists/${bookListId}`);
+      const response = await fetch(`/api/v1/bookLists`);
       debugger;
       if (!response.ok) {
         const errorMessage = `${response.status} (${response.statusText})`;
         const error = new Error(errorMessage);
         throw error;
       }
 
+      const bookListData = await response.json();
+      bookListData.bookLists.forEach((bookLists) => {
+        if (bookListId === bookLists.id) {
+          return setBookListName(bookLists.name);
+        }
+      });
+    } catch (err) {
+      console.error(`Error in fetch: ${err.message}`);
+    }
+  };
+  const getBooks = async () => {
+    try {
+      const response = await fetch(`/api/v1/books`);
+
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`;
+        const error = new Error(errorMessage);
+        throw error;
+      }
+
       const body = await response.json();
-      debugger;
-      setBooks(body.bookList.books);
-      setBookListName(body.bookList.name);
-      debugger;
+
+      setBooks(body.books);
```

Comment:
we shouldnt be making this fetch here, your original fetch should have everything you need